etc/spdk/iscsi.conf.in

```diff
@@ -99,11 +99,14 @@
 
   # The number of attempts per I/O when an I/O fails. Do not include
   # this key to get the default behavior.
-  NvmeRetryCount 4
-  # Registers the application to receive timeout callback and to reset the controller.
-  ResetControllerOnTimeout Yes
-  # Timeout value.
-  NvmeTimeoutValue 30
+  RetryCount 4
```

Comment:
I agree that these are better names, but we shouldn't arbitrarily break user configuration files without a good reason.